app/registries/foreman/access_permissions.rb

```diff
@@ -312,6 +312,10 @@
                                   }
   end
 
+  permission_set.security_block :host_statuses do |map|
+    map.permission :view_statuses, {:"api/v2/host_statuses" => [:index]}
```

Comment:
do we really need another permission for this? Isn't view_hosts the permission to allow this?

Reply:
it would make sense to use `view_hosts` permission 👍   just wondering, do we want to be able to grant permissions only to particular statuses?

Reply:
I would rather start without a specific permission and only add it if it is needed in the future, additional permissions add complexity and reduce performance.